### ISSUE TITLE
Tutorial 2 does not run on Windows 10 (python 3.7 64-bit)

### ISSUE THREAD

**Issue by BrendanSimon** [beeware/toga#742]
The tutorial 2 (box inside a box example) does not run on Windows 10 (python 3.7 64-bit).

I get the following error:

```
python -m box_in_box
[Winforms] Not implemented: Table.set_on_select()
Traceback (most recent call last):
  File "D:\Brendan\Temp\toga\venv\lib\site-packages\toga_winforms\app.py", line 120, in run_app
    self.create()
  File "D:\Brendan\Temp\toga\venv\lib\site-packages\toga_winforms\app.py", line 56, in create
    self.interface.startup()
  File "D:\Brendan\Temp\toga\venv\lib\site-packages\toga\app.py", line 177, in startup
    self.main_window.content = self._startup_method(self)
  File "D:\Brendan\Temp\toga\box_in_box.py", line 91, in build
    shortcut=toga.Key.MOD_1 + 'k',
  File "c:\Program Files\Python37\lib\enum.py", line 349, in __getattr__
    raise AttributeError(name) from None
AttributeError: MOD_1
```

https://toga.readthedocs.io/en/latest/tutorial/tutorial-2.html

Commenting out the line with `shortcut=toga.Key.MOD_1 + 'k'` allows the app to run.

```python
    cmd3 = toga.Command(
        action3,
        label='Action 3',
        tooltip='Perform action 3',
        #shortcut=toga.Key.MOD_1 + 'k',
        icon=cricket_icon
    )
```

**Comment by freakboy3742** [comment]
Thanks for the report! I'm not seeing this bug myself; can you clarify the Toga version/git hash that you are using? It's possible that you're seeing an inconsistency between the latest development code (which is what the published docs will be based on) and the most recent version posted to PyPI.

**Comment by freakboy3742** [comment]
This appears to be an error that was resolved in a4fff7b4. I've just released 0.3.0.dev15 that includes that patch.
